### ISSUE TITLE
"Too few arguments to block call, expected 3, have 1" when building

### ISSUE THREAD

**Issue by squgeim** [RevenueCat/react-native-purchases#32]
The following line throws an error when building:

https://github.com/RevenueCat/react-native-purchases/blob/bbc820b5106d40cab8e86157d3558abf89a9d85f/ios/RNPurchases.m#L138

The definition shows that it should have 3 arguments:

```objc
@property (nonatomic, copy) void (^reject)(NSString *code, NSString *message, NSError *error);
```

**Comment by anthonyjoeseph** [comment]
This seems like it has been fixed with version 1.5.1
https://github.com/RevenueCat/react-native-purchases/commit/51c5de03273ff549affa184d5241a7973520261c

**Comment by vegaro** [comment]
closing since it has been fixed
